### PR TITLE
Change tabs appearance

### PR DESCRIPTION
**Pull request by john-botella**
1 - Default
2 - Hover
3 - Active
4 - Improve badge visibility
new tab css

### CODE REVIEW

**Comment by eldy** [comment]
Is look and badge ok on smartphone too ?
On smartphone, the tabs are merged into a combo list.

**Comment by john-botella** [comment]
Good game ! I forgot the mobile testing part. Now it's fixed !
I added a note for me for the next time.

File: htdocs/theme/eldy/style.css.php [hunk 1]
```diff
@@ -2387,14 +2412,16 @@
...
+	margin-bottom: -1px !important;
 }
 a.tab:hover
 {
 	/*
 	background: rgba(<?php echo $colorbacktabcard1; ?>, 0.5)  url(<?php echo dol_buildpath($path.'/theme/'.$theme.'/img/nav-overlay3.png',1); ?>) 50% 0 repeat-x;
 	color: #<?php echo $colortextbacktab; ?>;
 	*/
-	text-decoration: underline;
+	text-decoration: none;
```
Comment (librethic-code) [inline htdocs/theme/eldy/style.css.php]: This is an accessibility error! Please keep in mind accessibility rules.
See https://www.sitepoint.com/15-rules-making-accessible-links/ (chapter 11) and https://www.w3.org/TR/WCAG20-TECHS/F73.html

**Comment by john-botella** [comment]
This is represented by top border of the tab.  
I added borders to make  tabs more "button like" ( for exemple  .butAction have text-decoration )

I'm agree with this type of recommandations for links in text paragraph, but we are not on  a text paragraph and it's not the link's normal state. 

The hover state change his border top line color saturation, hue and change cursor to pointer. 

If hover state style need more visibility for accessibility, I suggest changing hover state by the same of active state (remove bottom border and change top border to a 2px colored style. But actually I don't think it's necessary, don't you ?

**Comment by librethic-code** [comment]
You're right we are not into textual context. This point related to Dolibarr interface generally…

For tabs IMHO we need more contrast between active tabs and others (don't forget `:focus` state, using same styles than `:hover`), not only one border. Cursor changes is not valid when user navigate with keyboard with Tab key.

In any case, we applaud you for this courageous work on Dolibarr theming ;)

**Comment by john-botella** [comment]
Added :  more hover visual effect

**Comment by eldy** [comment]
Having borders on each tab gives a very loaded screen.
We retreive the look of tab we had in old version of Dolibarr before we removed the borders on tab to follow what bootstrap suggests (see screenshot). Also when we have several tabs on 2 lines (for example on small screen on tablet, we don't see anymore where is the badge (it is at both at top of a title but at bottom of another one. this is minor, this does not happen too frequently). So can you just add a constant so we can keep the "non selected tab" as simple links for people who want a cleaner interface ?

image

**Comment by john-botella** [comment]
yeah i see.

**Pull request closed without merging**